### PR TITLE
Enable Pytest in Candidate CI

### PR DESCRIPTION
**Pull request by blisc**

### CODE REVIEW

File: tests/collections/asr/test_asr_ctcencdec_model.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 class EncDecCTCModelTest(TestCase):
     @pytest.mark.unit
+    @pytest.mark.pleasefixme
```
Comment (titu1994) [inline tests/collections/asr/test_asr_ctcencdec_model.py:25]: We need a strong warning here when running tests on ci or locally that this test is being skipped.

File: Jenkinsfile [hunk 2]
```diff
@@ -33,11 +33,67 @@ pipeline {
       }
     }
 
+    stage('L0: Unit Tests GPU') {
+      steps {
+        sh 'pytest -m "unit and not skipduringci and not pleasefixme"'
```
[the remark this answers is not in the document]
Comment (blisc) [reply Jenkinsfile:38]: Yes these are gpu tests, CPU tests are done after.

File: tests/collections/asr/test_asr_ctcencdec_model.py (hunk 1 above)
Comment (blisc) [reply tests/collections/asr/test_asr_ctcencdec_model.py:25]: This is a very temporarily fix until candidate is stable.

**Comment by titu1994** [comment]
LGTM, please merge when ready

**Pull request merged**